device/core/src/pnp/properties.ts

```diff
@@ -0,0 +1,81 @@
+import { JSONValue, JSONObject } from '.';
+import { Twin } from '../twin';
+
+export function generateWritablePropertyResponse(value: JSONValue, ackCode: number, ackDescription: string, ackVersion: number): {
```

Comment:
We don't often have functions exported like this, should we move this somewhere else or does this seem fine?